### PR TITLE
Docs: Remove Incident and Recovery Threshold in the Documentation

### PR DESCRIPTION
**Pull request by haricnugraha**
# Monika Pull Request (PR)

## What feature/issue does this PR add
Remove incident and recovery threshold in the documentation

## How did you implement / how did you fix it
Search `incident` and `recovery` in the `docs` directory and remove it

## How to test
1. Go to `docs` directory. Run `cd docs`.
2. Install dependencies. Run `npm ci --legacy-peer-deps`.
3. Run the documentation website. Run `npm run dev`. 
4. Open `http://localhost:3000/` in your browser.

### CODE REVIEW

File: docs/src/pages/tutorial/write-config-in-visual-studio-code.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ If you have used Monika, you may notice that some keys are not there yet such as
 Now you can create your own Monika configuration using the methods above. For now, let’s try to create a Monika configuration based on the points below:
 
 - I want to use the Desktop notification channel
-- I want to create a single probe with the name “Google”, with any ID, has ten seconds interval, 10 times incidents/recoveries threshold
+- I want to create a single probe with the name “Google”, with any ID, has ten seconds interval
```
Comment (kevinhermawan) [inline docs/src/pages/tutorial/write-config-in-visual-studio-code.md:39]: ten seconds -> 10 seconds

**Review by kevinhermawan — APPROVED** [review]
Just small suggestions

File: docs/src/pages/tutorial/run-using-dingtalk-notification.md [hunk 2]
```diff
@@ -98,7 +96,7 @@ probes:
 Let me explain the configuration a little bit:
 
 - Monika will use DingTalk as their notification channel, using the access token from the previous step
-- Monika will probe [http://localhost:3000/api/demo/status](http://localhost:3000/api/demo/status) with recovery and incident threshold set to 1 every one second
+- Monika will probe [http://localhost:3000/api/demo/status](http://localhost:3000/api/demo/status) every one second
```
[the remark this answers is not in the document]
Comment (sapiderman) [reply docs/src/pages/tutorial/run-using-dingtalk-notification.md:99]: The convention is numbers less than ten is written out.  11 is written out as number.

**Review by sapiderman — APPROVED** [review]
Looks good! ready to merge!
📖 📜 📝

**Pull request merged**